snakewm/apps/games/maze3D/maze3d.py

```diff
@@ -1,13 +1,64 @@
 import pygame
 import os
 from math import cos, sin, tan, pi, floor, ceil, sqrt
-
+from random import randint
+import random
 two_pi = pi * 2
 half_pi = pi * 0.5
 three_half_pi = pi * 1.5
 
 NUMBER_OF_RAYS = 320
 
+class DisjointSet:
+	class Element:
+		def __init__(self, key):self.key = key;self.parent = self;self.rank = 0
+		def __eq__(self, other):return self.key ==  other.key
+		def __ne__(self, other):return self.key != other.key
+	def __init__(self):self.tree = {}
+	def make_set(self, key):
+		e = self.Element(key)
+		if not key in self.tree.keys():self.tree[key] = e
+	def find(self, key):
+		if key in self.tree.keys():element = self.tree[key]
+		if key in self.tree.keys() and element.parent != element:element.parent  = self.find(element.parent.key)
+		if key in self.tree.keys():return element.parent
+	def union(self, element_a, element_b):
+		root_a = self.find(element_a.key);root_b = self.find(element_b.key)
+		if root_a != root_b and root_a.rank < root_b.rank:root_a.parent = root_b
+		elif root_a != root_b and root_a.rank > root_b.rank:root_b.parent = root_a
+		elif root_a != root_b:root_b.parent = root_a;root_a.rank+=1
+
+class RandomMaze:
+	def __init__(self, width):self.width = width;self.height = width;self.seed = random.random()*10000;self.grid = [[(width*row + col) for row in range(0,width)] for col in range(0, width)];self.portals = {};self.kruskalize()
```

Comment:
This kind of line raises the question in me that what style guide a Snakeware source shall follow?
Do you use the Google Python Style Guide or enforce PEP8?

Reply:
Kinda bad decisions, but I do whatever I want

Reply:
We use Black for formatting, the style gets "auto-blacked" once it's merged into master. This looks cool though, I'll check it out soon!

Reply:
Oh. Thanks! Does not even know about Black till now. Today I learned something new. :)
I just questioned the format, as it is pretty hard to review long compact lines with no separators as they are not easy for the human eyes.

Reply:
Oops, sorry...